packages/sdk-components-react/src/radix-tooltip.ws.tsx

```diff
@@ -70,6 +80,22 @@ export const metaTooltip: WsComponentMeta = {
           component: "TooltipContent",
           label: "Tooltip Content",
           props: [],
+          /**
+           *  z-50 overflow-hidden rounded-md border bg-popover px-3 py-1.5 text-sm text-popover-foreground shadow-md
+           **/
+          styles: [
+            ...tc.preflight(),
```

Comment:
Maybe use object notation? Like { z: 50 }

Reply:
Or lists of classes like ['z-50']?

Reply:
Would be not a big deal to transform in following PRs. For now not stable at all

Reply:
or a `ts.merge(tc.preflight(), tc.z(50), ...)`

Reply:
Added `.flat()` native array method and seems like well typed with ts